examples/user_interaction/server/server.js

```diff
@@ -72,16 +72,24 @@ function handleRequest(request, response) {
 
     // Necessary headers because the Node.js and React dev servers run in different ports.
     response.setHeader('Access-Control-Allow-Origin', '*');
-    response.setHeader('Access-Control-Allow-Methods', 'GET, POST, OPTIONS, PUT, PATCH, DELETE');
-    response.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,content-type');
+    response.setHeader('Access-Control-Allow-Methods', 'GET, OPTIONS');
+    response.setHeader('Access-Control-Allow-Headers', 'traceparent');
     response.setHeader('Access-Control-Allow-Credentials', true);
 
     let result = '';
     let code = 200;
+    if(request.method !== 'GET'){
```

Comment:
Is this logic for CORS preflght requests? If so, could you add a comment to explain that (and maybe match `request.method === 'OPTIONS'`)?